projects/packages/jetpack-mu-wpcom/src/class-jetpack-mu-wpcom.php

```diff
@@ -94,16 +92,12 @@ public static function load_features() {
 
 		// Initializers, if needed.
 		\Marketplace_Products_Updater::init();
+		\Jetpack_Calypsoify::get_instance();
```

Comment:
Either bring back the `use` or do `\Automattic\Jetpack\Calypsoify\Jetpack_Calypsoify` here.